### DISCUSSION TITLE
Remove peak coordinates in markerHeatmap

### DISCUSSION THREAD

**Discussion by yuanlizhanshi** [GreenleafLab/ArchR#1473] [Questions / Documentation] [answered]
When i run the tutorial code.
`heatmapPeaks <- markerHeatmap( seMarker = markersPeaks, cutOff = "FDR <= 0.1 & Log2FC >= 0.5", transpose = TRUE ) draw(heatmapPeaks, heatmap_legend_side = "bot", annotation_legend_side = "bot")`
The heatmap was different from the tutorial.
How to remove the peak coordinates of the heatmap
![173531221-06d7a612-1ba3-4f26-ab24-42103aa29983](https://user-images.githubusercontent.com/38498877/173602259-c5c7c9d3-3b10-425f-b8d6-d32428da65ed.png)
.
I have checked the usage of this [plotMarkerHeatmap](https://www.archrproject.com/reference/plotMarkerHeatmap.html) carefully but still can't remove this.

**Comment by rcorces** [comment, marked as answer]
<html>
<body>


nLabel | An integer value that indicates whether the top n features for each column in seMarker should be labeled on the side of the heatmap.
-- | --



</body>
</html>

Set `nLabel = 0`

**Reply by yuanlizhanshi** [reply]
Worked, thank you very much
